### PR TITLE
Added $opts array for customize gRPC client options

### PR DESCRIPTION
**Pull request by aneterial**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=ru

## Pull request type

Please check the type of change your PR introduces:

- [x] Feature

## What is the current behavior?

Issue Number: [#136](https://github.com/ydb-platform/ydb-php-sdk/issues/136)

## What is the new behavior?

You can customize the gRPC client's behavior by setting options in config array

Example of using:
```php
$config = [
    // ...
    'opts' => [
        'grpc.max_receive_message_length' => 8*1024*1024,
        'grpc.default_compression_algorithm' => 2,
        'grpc.default_compression_level' => 2,
    ],
]
$ydb = new \YdbPlatform\Ydb\Ydb($config);

### CODE REVIEW

File: src/Ydb.php [hunk 1]
```diff
@@ -173,6 +179,14 @@ public function meta(): array
         return $meta;
     }
 
+    public function opts(): array
+    {
+        $opts = $this->opts;
+        $opts['credentials'] = $this->iam()->getCredentials();
```
Comment (rekby) [inline src/Ydb.php:185]: At the place need to clone original dict `$this->opts` and change copy or move `$opts['credentials'] = $this->iam()->getCredentials();` to initialization phase for prevent unexpected change the dict.

File: src/Ydb.php [hunk 2]
```diff
@@ -106,6 +111,7 @@ public function __construct($config = [], LoggerInterface $logger = null)
         $this->endpoint = $config['endpoint'] ?? null;
         $this->database = $config['database'] ?? null;
         $this->iam_config = $config['iam_config'] ?? [];
+        $this->opts = $config['opts'] ?? [];
```
Comment (rekby) [inline src/Ydb.php:114]: better to clone opts from config

File: README.md [hunk 3]
```diff
@@ -562,3 +562,19 @@ $config = [
...
 
+## gRPC options
+
+You can customize the gRPC client's behavior by setting options in config array
+
+Example of using:
+```php
+$config = [
+    // ...
+    'opts' => [
```
Comment (rekby) [inline README.md:573]: it will be better to separate grpc options to own section: 'grpc' instead of common 'opts'

**Review by rekby — CHANGES_REQUESTED** [review]
Hello @aneterial.

Thanks for the PR.
Great job!

I have few small comments for it.

File: README.md (hunk 3 above)
Comment (aneterial) [reply README.md]: fixed

**Review by rekby — APPROVED** [review]

**Comment by rekby** [comment]
@aneterial thanks for the PR :)

**Pull request merged**